cfe_internal/update/simple_policy_update_v1.cf

```diff
@@ -0,0 +1,476 @@
+################################################################################
```

Comment:
This file is the old update_policy.cf , and I renamed cfe_internal_update_policy to simple_policy_update_v1_cfe_internal_update_policy  ( I was lazy and just prefixed it for now, better name welcome)